packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -128,10 +141,7 @@ class _AllocationTracingTableState extends State<AllocationTracingTable> {
             labelText: 'Class Filter',
             hintText: 'Filter by class name',
             onChanged: (value) {
-              ga.select(
-                analytics_constants.memory,
-                analytics_constants.MemoryEvent.tracingClassFilter,
-              );
+              _sendFilterEditGaEvent();
```

Comment:
is there a reason we can't use onSubmitted or onEditingComplete? Seems a bit hack to throttle when we can use one of the text field lifecycle methods that indicates "the user is done editing"

Reply:
both are designed to handle ui forms or dialogs that at some moment are 'done'. they need special signal that editing is done. it is not our case. 

throttling seems to be simplest and most reliable way to handle ga for filter

https://stackoverflow.com/questions/63690311/flutter-textfield-difference-between-onedittingcomplete-and-onsubmitted